tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -280,30 +280,26 @@ max-allowed-packet = 67_108_864
 # 此服务的私钥。默认为 `security.key-path` 的副本
 # key-path = "/path/to/lightning.key"
 
-# 数据导入完成后，tidb-lightning 可以自动执行 Checksum、Compact 和 Analyze 操作。
+# 对于 Local/Importer Backend 模式，数据导入完成后，tidb-lightning 可以自动执行 Checksum 和 Analyze 操作。
 # 在生产环境中，建议这将些参数都设为 true。
-# 执行的顺序为：Checksum -> Compact -> Analyze。
+# 执行的顺序为：Checksum -> Analyze。
+# 注意：对于 TiDB Backend, 无须执行这两个阶段，因此在实际运行时总是会直接跳过。
 [post-restore]
 # 配置是否在导入完成后对每一个表执行 `ADMIN CHECKSUM TABLE <table>` 操作来验证数据的完整性。
 # 可选的配置项：
 # - "required"（默认）。在导入完成后执行 CHECKSUM 检查，如果 CHECKSUM 检查失败，则会报错退出。
 # - "optional"。在导入完成后执行 CHECKSUM 检查，如果报错，会输出一条 WARN 日志并忽略错误。
 # - "off"。导入结束后不执行 CHECKSUM 检查。
 # 默认值为 "required"。从 v4.0.8 开始，checksum 的默认值由此前的 "true" 改为 "required"。
-# 注意：考虑到与旧版本的兼容性，依然可以在本配置项设置 `true` 和  `false` 两个布尔值，其效果与 "required" 和 `off` 相同。
+#
+# 注意：
+# 1. Checksum 对比失败通常表示导入异常(数据丢失或数据不一致)，因此建议总是开启 Checksum。
+# 2. 考虑到与旧版本的兼容性，依然可以在本配置项设置 `true` 和 `false` 两个布尔值，其效果与 "required" 和 `off` 相同。
 checksum = "required"
 # 配置是否在 CHECKSUM 结束后对所有表逐个执行 `ANALYZE TABLE <table>` 操作。
 # 此配置的可选配置项与 `checksum` 相同，但默认值为 "optional"。
 analyze = "optional"
 
-# 如果设置为 true，会在导入每张表后执行一次 level-1 Compact。
-# 默认值为 false。
-level-1-compact = false
-
-# 如果设置为 true，会在导入过程结束时对整个 TiKV 集群执行一次 full Compact。
-# 默认值为 false。
-compact = false
```

Comment:
Why delete the description of these compact parameters ?

Reply:
Because they are out of date and no used anymore. For any tidb cluster >= v4.0.0, it's not recommend for the user to explicitly trigger rocksdb compaction anymore.